perf.groovy

```diff
@@ -88,9 +88,9 @@ def static getOSGroup(def os) {
                         batchFile("\"%WORKSPACE%\\nuget.exe\" install Microsoft.BenchView.JSONFormat -Source http://benchviewtestfeed.azurewebsites.net/nuget -OutputDirectory \"%WORKSPACE%\" -Prerelease -ExcludeVersion")
                         //Do this here to remove the origin but at the front of the branch name as this is a problem for BenchView
                         //we have to do it all as one statement because cmd is called each time and we lose the set environment variable
-                        batchFile("if [%GIT_BRANCH:~0,7%] == [origin/] (set GIT_BRANCH_WITHOUT_ORIGIN=%GIT_BRANCH:origin/=%) else (set GIT_BRANCH_WITHOUT_ORIGIN=%GIT_BRANCH%)\n" +
-                        "set BENCHVIEWNAME=${benchViewName}\n" +
-                        "set BENCHVIEWNAME=%BENCHVIEWNAME:\"=%\n" +
+                        batchFile("if [%GIT_BRANCH:~0,7%] == [origin/] (set \"GIT_BRANCH_WITHOUT_ORIGIN=%GIT_BRANCH:origin/=%\") else (set \"GIT_BRANCH_WITHOUT_ORIGIN=%GIT_BRANCH%\")\n" +
```

Comment:
Should we use "double quotes" instead of [square brackets] in the if condition? Otherwise we'll break here when expanding that, I think.